portal/src/pages/index.vue

```diff
@@ -31,10 +31,19 @@
           />
         </div>
       </div>
-      <div class="center_block main__thems_and_communities" v-show="false">
+
+      <div class="main__materials">
+        <div class="center_block">
+          <h2 class="main__materials_title">{{ $t('Newest-open-learning-material') }}</h2>
+          <Materials :materials="materials" />
+        </div>
+      </div>
+
+      <div class="center_block main__thems_and_communities">
         <Themes
           :themes="sortedThemes"
           class="main__thems"
+          v-show="false"
```

Comment:
Ik zou het gewoon weghalen. Of komt het ook weer tevoorschijn op de 1 of andere manier?

Reply:
Ik vind het prima om het weg te halen, alleen begreep ik van Fako dat ik het moest bewaren omdat het er later weer opkomt.

Reply:
Het is al een component toch? Dus we moeten alleen het component terugplaatsen. Ik ben persoonlijk niet zo'n voorstander van dingen laten staan omdat ze mogelijk ooit weer terug zouden komen. Daar hebben we git voor

Reply:
ik heb het weggehaald

Reply:
In principe wel mee eens om te verwijderen, maar als ik alles had weggehaald ... hoe snel had een programmeur zonder kennis van het project dan in GIT kunnen terugvinden hoe communities component werd gebruikt? Als je verwacht zeer binnenkort weer iets nodig te hebben, dan is "uitschakelen" soms praktischer. Thema's is een beetje onduidelijk wat er mee gaat gebeuren. Sommige mensen bij SURF vinden dit ERG belangrijk.

Als we makkelijker GIT history willen lezen dan helpt het om "nog een test" commits te vermijden. PR's squashen is ook een goed idee. Dingen waarvan we zeker zijn dat ze niet terug komen (zoals Edurep search), moeten we zeker gewoon uit code verwijderen. Liefst met een vermelding in een release in Github, zodat als we ooit eens willen kijken niet de volledige history hoeven na te pluizen, maar alleen releases lijstje.

Reply:
Ik bedoelde het stuk in de HTML weg, niet het hele component weggooien.

Reply:
De frontend code is best slecht geschreven en niet gedocumenteerd. Het gebruik van een component is vaak de enige indicatie van hoe zo'n component bedacht is. Vooral met allerlei vage props waarvan ik geen idee heb wat ze doen. Ik heb nog gecheckt met Jelmer, maar voorlopig houden we alle thema code. Je kunt het gebruik op home nu wel verwijderen. Ik maak even een aantekening in de releases. De header heeft ook nog allerlei thema referenties.

Reply:
Ja, dat was mij ook niet ontgaan haha. Misschien is het dan maar beter dat we het niet hergebruiken 😝